qa-mickey.planx-pla.net/manifest.json

```diff
@@ -57,7 +57,7 @@
     "sync_from_dbgap": "False",
     "useryaml_s3path": "s3://cdis-gen3-users/qa/user.yaml",
     "dispatcher_job_num": "10",
-    "netpolicy": "off"
+    "netpolicy": "on"
```

Comment:
@cterrazas2 Will the change effect mariner?

Reply:
@jingh8 Sorry just saw this! This is fine, Fasi had worked on adding Mariner to gen3 roll all. So netpolicy "on" is fine. Here is the [PR](https://github.com/uc-cdis/cloud-automation/pull/1554) Fasi worked on.

Reply:
looks like that PR was never merged?

Reply:
Good catch @paulineribeyre , well that's conflicting as the Jira ticket says done. So, I rolled mariner with netpolicy "on" in qa-mickey and it skips the netpolicy step and spits out a "not accessible by annotation acl" log, but mariner still deploys and I can run a workflow? I'm not quite sure why that PR was closed, will investigate.